source/documentation/github/create-project.md

```diff
@@ -0,0 +1,79 @@
+# Create a new project in GitHub
+
+In GitHub, you can use a repository to store and collaborate on all of your project's code.
+
+You can also use:
+
+- [pull requests](https://docs.github.com/en/pull-requests) to propose changes to your code and receive reviews from others
+- [issues](https://docs.github.com/en/issues) to plan and track your work
+
+### Create a new repository
+
+To create a new repository, you should:
+
+1. Navigate to the [MoJ Analytical Services](https://github.com/moj-analytical-services) GitHub organisation page.
+2. In the repositories section, select **New**.
+3. Configure the properties of your repository, in line with the information below.
+4. Select **Create repository**.
+
+#### Owner
+
+You should select **moj-analytical-services** as the **Owner** of the repository. This should be selected by default.
+
+If you accidentally create a repository in your personal GitHub account or a different organisation, you can [transfer it](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account) to the MoJ Analytical Services GitHub organisation.
+
+#### Repository name
+
+When choosing a name for your repository, we recommend:
+
+- keeping it short and simple
+- making it descriptive and meaningful
+- avoiding abbreviations and acronyms
+- using dashes to separate words
```

Comment:
An extra line to say "Use dashes not full stops in repo name" ?

Reply:
Updated in 6cd7aa6